docs/user/private-app-repository.md

```diff
@@ -34,6 +34,35 @@ When Kubeapps deploys any chart from this AppRepository, if a referenced docker
 
 There will be further work to enable private AppRepositories to be available in multiple namespaces. Details about the design can be read on the [design document](https://docs.google.com/document/d/1YEeKC6nPLoq4oaxs9v8_UsmxrRfWxB6KCyqrh2-Q8x0/edit?ts=5e2adf87).
 
+## Filter applications
+
+Since Kubeapps 2.3, it's possible to limit the amount of packages that Kubeapps will expose to the users of an Application Repository. For doing so, edit the filter field of the form:
+
+<img src="../img/apprepo-form-filter.png" alt="AppRepository with filter" width="600px">
+
+- If "Exclude Packages" is marked, all the applications matching the given names will be excluded.
+- If "Regular Expression" is marked, rather than treating the input as a comma-separated list of application names, it will be treated as a regex. This can be used when the name of the application is not known or when you want to filter different applications with the same name.
```

Comment:
I'd rather say `(...) treated as a javascript regex` since some minor things are not supported in js regex (such as recursion) but they are in PCRE.

Reply:
TBH, I am not sure what type of regexes jq uses (probably regex, but I would need to check).

Reply:
so they are PCRE regexes: https://stedolan.github.io/jq/manual/#RegularexpressionsPCRE

Reply:
Great, thanks for checking it :)